Update guide on Ubuntu 21.10 Firefox snap changes (#31255)
dotnet/AspNetCore.Docs: aspnetcore/security/enforcing-ssl.md

}
EOF
```

> Quick note: Since Ubuntu 21.10 Firefox comes as a snap package, so the new installation folder is : `/snap/firefox/current/usr/lib/firefox`.

Reviewer: ```suggestion
Note: Ubuntu 21.10 Firefox comes as a snap package and the installation folder is `/snap/firefox/current/usr/lib/firefox`.
```
